tests/types/ember-concurrency-test.ts

```diff
@@ -714,58 +1217,76 @@ module('unit tests', () => {
     }
 
     {
-      let tp = task({ *perform() {} });
-      expect(tp).toEqualTypeOf<TaskProperty<void, []>>();
+      let tp = task({ foo: 'foo', *perform() {} });
+      expect(tp).toMatchTypeOf<TaskProperty<void, []>>();
```

Comment:
The `toMatchTypeOf` here and in the rest of the file confirms that this is non-breaking, since it's still assignable.